dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -248,6 +248,15 @@
 			status = "disabled";
 			label = "UART_8";
 		};
+		uart9: serial@40011800 {
+			compatible = "st,stm32-uart";
+			reg = <0x40011800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000040>;
+			interrupts = <155 0>;
+			current-speed = <115200>;
```

Comment:
`current-speed` seems unnecessary.

Reply:
yes I overlooked it, will remove that line

Reply:
yes I overlooked it, will remove that line

Reply:
Update I have force push the fix